assets/js/state/selectors/cluster.test.js

```diff
@@ -24,6 +28,41 @@ describe('Cluster selector', () => {
     expect(getClusterHostIDs('cluster1')(state)).toEqual(['id1', 'id2']);
   });
 
+  it('should return the cluster hosts names', () => {
+    const state = {
+      hostsList: {
+        hosts: [
```

Comment:
Hey what about using the hostFactory for that?

Something like this ? 
```
hosts: [
      hostFactory.build({ params: { id: 'id1', cluster_id: 'cluster1', hostname: 'hostname-1' } }),
      hostFactory.build({ params: { id: 'id2', cluster_id: 'cluster1', hostname: 'hostname-2' } }),
      hostFactory.build({ params: { id: 'id3', cluster_id: 'cluster2', hostname: 'hostname-3' } }),
    ]
```
    
You can find the the host factory here:
`assets/js/lib/test-utils/factories/hosts.js`

Reply:
Sure, thanks!